### ISSUE TITLE
Create source for SEPA

### ISSUE THREAD

**Issue by armetiz** [stripe/react-stripe-elements#41]
Hi there;

I'm using this component to create source for cards.

Could I use the `injectStripe` to get configured Stripe and create source for SEPA Debits ?

Right now; I'm getting the following error : 

```
You did not specify the type of Source or Token to create.
We could not infer which Element you want to use for this operation.
```

**Comment by michelle** [comment]
Hi! This bug is fixed in master. I'll do a release today.

**Comment by jenan-stripe** [comment]
Released 0.0.3!
